### DISCUSSION TITLE
How to make diagrams in GitHub flavoured markdown

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/sa4ss#99] [Show and tell]
I was struggling to come up with a topic for this weeks show and tell until I came across a blog post on [how to use 🧜‍♀️ on :octocat:](https://andrewlock.net/drawing-graphs-in-github-comments-with-mermaid-diagrams/). The result is we can create things like
```mermaid
flowchart LR
    A[Hard] -->|Text| B(Round)
    B --> C{Decision}
    C -->|One| D[Result 1]
    C -->|Two| E[Result 2]
```
in :octocat: issues, pull requests, and discussions. More information on 🧜‍♀️ can be found on the :octocat: [README for 🧜‍♀️](https://github.com/mermaid-js/mermaid#about). I think I am most interested in learning/becoming proficient at the creation of gantt charts to note deadlines for milestones but maybe this is repetitive with the use of project boards, which I do not seem to be very good at.
```mermaid
gantt
    section Section
    Completed :done,    des1, 2014-01-06,2014-01-08
    Active        :active,  des2, 2014-01-07, 3d
    Parallel 1   :         des3, after des1, 1d
    Parallel 2   :         des4, after des1, 1d
    Parallel 3   :         des5, after des3, 1d
    Parallel 4   :         des6, after des4, 1d
```

Please feel free to 💬 on anything that pops into your 🤯 but I am particularly interested in knowing when and where we can use gantt charts to help increase our productivity and people's knowledge of what is coming down the pipeline to streamline the development process.

**Comment by iantaylor-NOAA** [comment]
These diagrams are amazing. 

One frustration I have is that lots of the good stuff in github markdown (including these diagrams) doesn't get translated very well into emails and I initially read most github notifications by email and then click the link to "view it on GitHub" if I want to respond. Perhaps I should get in the habit of looking at https://github.com/issues instead, but that wouldn't show me notifications of discussions like this.
